platform_umbrella/apps/home_base_web/priv/raw_files/install_script.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env sh
+
+set -eux
+
+BI="bi"
+TMP_DIR="$(mktemp -d)"
+
+INSTALL_SPEC_URL="<%= spec_url %>"
+
+trap 'rm -rf "${TMP_DIR}"' INT TERM EXIT
+
+check_installed() {
+    command -v "${1}" >/dev/null 2>&1
+}
+
+install_bi() {
+    REPO="batteries-included/batteries-included"
+    INSTALL_DIR="$HOME/.local/bin"
+    OS="$(uname -s)"
+    ARCH="$(uname -m)"
+    EXT=".tar.gz"
+    OUTPUT_LOC="${TMP_DIR}/${BI}${EXT}"
+
+    VERSION=$(curl --silent "https://api.github.com/repos/${REPO}/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")')
+    DOWNLOAD_URL=$(curl --silent "https://api.github.com/repos/${REPO}/releases/tags/${VERSION}" | grep "browser_download_url.*${OS}_${ARCH}${EXT}" | cut -d '"' -f 4)
+
+    if [ -z "${DOWNLOAD_URL}" ]; then
+        echo "Failed to find the asset for version ${VERSION}."
+        exit 1
+    fi
+
+    echo "Downloading ${DOWNLOAD_URL} ..."
+    curl -L -o "${OUTPUT_LOC}" "${DOWNLOAD_URL}"
+
+    # extract to temp directory
+    if [ "$EXT" = ".tar.gz" ]; then
+        tar -xf "${OUTPUT_LOC}" -C "${TMP_DIR}"
+    elif [ "$EXT" = ".zip" ]; then
+        unzip "${OUTPUT_LOC}" -d "${TMP_DIR}"
+    fi
+
+    chmod +x "${TMP_DIR}/${BI}"
+    mv "${TMP_DIR}/${BI}" "${INSTALL_DIR}/${BI}"
```

Comment:
What do you think about putting the bi binary in ~/.local/share/<<VERSION>> and then linking the most recent version? That would allow atomic updates later and would allow some versioning.